DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/TopCourses/Index.cshtml

```diff
@@ -0,0 +1,80 @@
+﻿@inject IConfiguration configuration
+
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Centre.TopCourses
+@using Microsoft.Extensions.Configuration
+@model TopCoursesViewModel
+
+
+@{
+  ViewData["Title"] = "Top courses";
+  ViewData["Application"] = "Tracking System";
+  ViewData["HeaderPath"] = $"{configuration["AppRootPath"]}/TrackingSystem/Centre/Dashboard";
+  ViewData["HeaderPathName"] = "Tracking System";
+}
+
+@section NavMenuItems {
+  <partial name="~/Views/TrackingSystem/Shared/_NavMenuItems.cshtml" />
+}
+
+@section NavBreadcrumbs {
+  <partial name="~/Views/TrackingSystem/Shared/_CentreDashboardBreadcrumbs.cshtml"/>
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <h1 class="nhsuk-heading-xl">Top courses</h1>
+
+    <table role="table" class="nhsuk-table-responsive">
+      <thead role="rowgroup" class="nhsuk-table__head">
+      <tr role="row">
+        <th role="columnheader" class="nhsuk-u-padding-left-2" scope="col">
+          Name
+        </th>
+        <th role="columnheader" class="nhsuk-u-padding-left-2" scope="col">
+          Delegates
+        </th>
+        <th role="columnheader" class="nhsuk-u-padding-left-2" scope="col">
+          In progress
+        </th>
+        <th role="columnheader" class="nhsuk-u-padding-left-2" scope="col">
+          Completed
+        </th>
+        <th role="columnheader" class="nhsuk-u-padding-left-2" scope="col">
+          Pass rate
+        </th>
+        <th role="columnheader" class="nhsuk-u-padding-left-2" scope="col">
+          Action
+        </th>
+      </tr>
+      </thead>
+      <tbody class="nhsuk-table__body">
+      @foreach (var course in Model.TopCourses) {
+        <tr role="row" class="nhsuk-table__row">
+          <td role="cell" class="nhsuk-table__cell nhsuk-u-padding-left-2 cell-right-padding">
+            <span class="nhsuk-table-responsive__heading">Name </span>@course.CourseName
+          </td>
+          <td role="cell" class="nhsuk-table__cell nhsuk-u-padding-left-2 cell-right-padding">
+            <span class="nhsuk-table-responsive__heading">Delegates </span>@course.DelegateCount

```

Comment:
These columns display just a number (see screen shot), so when read out with a screen reader, you typically hear something like "Delegates Column Two 187" which might add some confusion with the column number and data being read aloud. 

This is also an issue in the 3 column table example on the NHS component, so I don't think anything needs to be done if they consider it satisfactory, but it's worth bearing in mind. Link to component: https://service-manual.nhs.uk/design-system/components/table#3-or-more-column-table